pkg/model/base.go

```diff
@@ -15,11 +15,11 @@ type BaseModel struct {
 }
 
 // BeforeCreate prepare data before create data
-func (m *BaseModel) BeforeCreate(tx *gorm.DB) (err error) {
-	m.ID = NewUUID()
-	m.CreatedAt = time.Now()
-	return
-}
+// func (m *BaseModel) BeforeCreate(tx *gorm.DB) (err error) {
+// 	m.ID = NewUUID()
+// 	m.CreatedAt = time.Now()
+// 	return
+// }
```

Comment:
I'm not sure but I think it's a mistake, or is there any purpose for it ?

Reply:
it's not necessary but we still want to keep it